src/quo2/components/settings/settings_list/view.cljs

```diff
@@ -99,21 +116,22 @@
            badge?
            button-props
            communities-props
-           container-style]}]
+           container-style
+           override-theme
+           status-tag-props]}]
   [rn/touchable-without-feedback
-   {:on-press            on-press
+   {:on-press            (when on-press on-press)
```

Comment:
Be careful, `:on-press` receives a function, and we are passing `on-press` when it exists or `nil` when it doesn't, and it's the same as before this change.

I mean, before:
```clojure
{:on-press on-press
 ,,,
}
```

Now:


```clojure
{:on-press (when on-press on-press)
 ,,,
}
```